bin/build_cmd.mli

```diff
@@ -1,5 +1,10 @@
 open Import
 
+val build_via_rpc_server
+  :  print_on_success:bool
+  -> Dune_lang.Dep_conf.t list
+  -> unit Fiber.t
```

Comment:
Now might be a good time to document that export ? You could move the implementation comment to use it as a basis.